language/Portuguese/strings.po

```diff
@@ -1040,1156 +1083,1182 @@ msgid ""
 "%s\n"
 " removed from favorites list."
 msgstr ""
+"%s\n"
+" removido da lista de favoritos."
 
 #: src/frontend/mame/ui/selgame.cpp:441
 msgid "Configure Machine"
-msgstr ""
+msgstr "Configurar Máquina"
 
 #: src/frontend/mame/ui/selgame.cpp:445 src/frontend/mame/ui/miscmenu.cpp:908
 msgid "Plugins"
-msgstr ""
+msgstr "Plugins"
 
 #: src/frontend/mame/ui/selgame.cpp:853
 #, c-format
 msgid "Romset\t%1$-.100s\n"
-msgstr ""
+msgstr "Romset\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:854
 #, c-format
 msgid "Year\t%1$s\n"
-msgstr ""
+msgstr "Romset\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:855
 #, c-format
 msgid "Manufacturer\t%1$-.100s\n"
-msgstr ""
+msgstr "Fabricante\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:859
 #, c-format
 msgid "Driver is Clone of\t%1$-.100s\n"
-msgstr ""
+msgstr "Driver é Clone de\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:861
 msgid "Driver is Parent\t\n"
-msgstr ""
+msgstr "Driver é Original\t\n"
```

Comment:
Using "Original" as a translation for MAME's concept of a "Parent" set is misleading.  The parent/clone distinction is somewhat arbitrary.  The parent set is generally the most recent version of the English-language release with most widespread distribution.  In many cases this is not the original version.  For example, the original version of Capcom's fighting games would be the Japanese releases, but the English world releases are the parent sets in MAME.  Can you think of a term that doesn't imply the parent set is the original version?

Reply:
You're right, "Original" is not really adequate.
Not an easy translation to do in this context. The more direct translations, such as "Pai" would have a meaning closer to "Father", and are not really the best either.
After some consideration, decided to go with "Principal" (something like "Main" in English).